components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```
@@ -7855,6 +7855,15 @@ components:
         type:
           type: string
           example: HTTP
+        audience:
+          type: string
+          description: The audience of the API. Accepted values are PUBLIC, PRIVATE, SINGLE, PARTNERS
+          example: PUBLIC
+          enum:
+            - PUBLIC
+            - PRIVATE
+            - SINGLE
+            - PARTNERS
```

Shall we add only the implemented one to the API?
We can keep all 4 in the internal classes.
```suggestion
            - PUBLIC
            - SINGLE
```

---

Removed